aggregators/1.0/modules/ROOT/pages/aggregators-cluster.adoc

```diff
@@ -0,0 +1,21 @@
+= Considerations for Using Aggregators in a Cluster
+
+[IMPORTANT]
+Aggregators don’t work with multiple workers and a persistent object store because clustering is not available in CloudHub. For details on how workers can be shared or doubled to scale your application and provide high availability, see the xref:runtime-manager::cloudhub-fabric.adoc[CloudHub Fabric] documentation.
+
+Aggregators Module works in a cluster out-of-the-box environment. However, to prevent an unexpected behavior consider the following:
+
+When an asynchronous aggregation is defined and the first element arrives, it is scheduled in the primary node of the cluster.
+Because new values can arrive in any node of the cluster, notify and make the primary node schedule that aggregation. To do that, another task in the primary node periodically determines whether it is necessary to schedule a new aggregation or not. This can lead to a problem if the interval between checks for new aggregation scheduling is much longer than the actual timeout of the aggregation, because the aggregation could be over before it is scheduled, or there might be errors in the time computation.
```

Comment:
The second sentence is missing something.
And the narrative/chronology here is unclear, at least to me.
It seems that we are saying that in a cluster, the primary node is responsible for scheduling aggregations, but because new data can arrive at any node in the cluster, it must periodically scan all nodes for that new data to aggregate, and if those scans are too infrequent, the aggregation might time out before all the data can be collected.
I also don't understand the part about errors in time computation...which seems to be different words for the same thing, a mismatch between the frequency of data scanning and the scheduled aggregation timeout.